### PR TITLE
adds run instructions with the wercker cli

### PR DESCRIPTION
**Pull request by faiq**
I didn't know how to run the application at first when I cloned it locally. So this adds a sentence on how to quickly run the application in dev mode.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ getting-started-python
 Python sample application for wercker leveraging the Flask
 microframework
 
+To run the application make sure you have the [wercker cli installed with it's proper requirements](http://devcenter.wercker.com/learn/basics/the-wercker-cli.html).
+
+Once you have all that set up, you can start the application by running `wercker dev --publish 5000`. This will run your application inside your Docker host, which you can visit on `192.168.99.100:5000` if you're using `docker-machine` or `192.168.59.103:5000` if you're using `boot2docker`. If the address of your docker container isn't one of the ones we mentioned earlier, the `wercker dev` command will tell you the address you can find your application on.
```
Comment (hatchan) [inline README.md]: If `wercker dev` show the correct ip, then we might as well remove the other ip's to make it less confusing.

Comment (hatchan) [inline README.md]: Also, could you add some new lines, makes it easier to make comments, and to work with vim

**Comment by hatchan** [comment]
Minor comment, :+1:

**Comment by termie** [comment]
lgtm

File: README.md [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
 
+[![Wercker
+status](https://app.wercker.com/status/b312ecb5c6fdd7c6eb871455a5b8964e/s)](https://app.wercker.com/project/bykey/b312ecb5c6fdd7c6eb871455a5b8964e)
+
 Python sample application for wercker leveraging the Flask
 microframework
 
-[![Wercker
-status](https://app.wercker.com/status/b312ecb5c6fdd7c6eb871455a5b8964e/s)](https://app.wercker.com/project/bykey/b312ecb5c6fdd7c6eb871455a5b8964e)
+To run the application make sure you have the [wercker cli installed with it's proper requirements](http://devcenter.wercker.com/learn/basics/the-wercker-cli.html).
```
Comment (mwhooker) [inline README.md]: to run the application, make sure

**Comment by mwhooker** [comment]
comments, lgtm

**Pull request merged**